include/aspect/material_model/rheology/visco_plastic.h

```diff
@@ -107,6 +107,20 @@ namespace aspect
        * The current cohesion.
        */
       std::vector<double> current_cohesions;
+
+      /**
+       * The LHS term corresponding to plastic dilation in the
+       * Stokes system. For details, see the comments of
+       * MaterialModel::PrescribedDilation::dilation_lhs_term.
+       */
+      std::vector<double> dilation_lhs_terms;
+
+      /**
+       * The RHS term corresponding to plastic dilation in the
+       * Stokes system. For details, see the comments of
+       * MaterialModel::PrescribedDilation::dilation_rhs_term.
+       */
+      std::vector<double> dilation_rhs_terms;
```

Comment:
this code part looks different since #6384. Please rebase, the new version will be simpler, because it includes the `DruckerPragerParameters` structure already, which will automatically include the dilation angle.